kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -206,12 +290,66 @@ <h4 class="mzp-c-menu-item-title">{{ _('Browse by product') }}</h4>
               <h4 class="mzp-c-menu-item-title">{{ _('Browse all forum threads by topic') }}</h4>
             </div>
             <ul class="mzp-c-menu-item-list sumo-nav--sublist">
-              <li><a href="{{ url('questions.list_by_topic', topic_slug='troubleshooting') }}">{{ _('Troubleshooting') }}</a></li>
-              <li><a href="{{ url('questions.list_by_topic', topic_slug='protect-your-privacy') }}">{{ _('Protect your privacy') }}</a></li>
-              <li><a href="{{ url('questions.list_by_topic', topic_slug='manage-preferences-and-add-ons-firefox') }}">{{ _('Customize settings and preferences') }}</a></li>
-              <li><a href="{{ url('questions.list_by_topic', topic_slug='install-and-update') }}">{{ _('Install and update') }}</a></li>
-              <li><a href="{{ url('questions.list_by_topic', topic_slug='install-and-manage-add-ons-firefox') }}">{{ _('Install and manage add-ons') }}</a></li>
-              <li><a href="{{ url('questions.list_by_topic', topic_slug='emails-thunderbird') }}">{{ _('Emails') }}</a></li>
+              <li>
+                <a href="{{ url('questions.list_by_topic', topic_slug='troubleshooting') }}"
+                   data-event-name="link_click"
+                   data-event-parameters='{
+                    "link_name": "main-menu.question-list",
+                    "link_detail": "troubleshoooting"
+                  }'>
+                   {{ _('Troubleshooting') }}
+                </a>
+              </li>
+              <li>
+                <a href="{{ url('questions.list_by_topic', topic_slug='protect-your-privacy') }}"
+                   data-event-name="link_click"
+                   data-event-parameters='{
+                    "link_name": "main-menu.question-list",
+                    "link_detail": "protect-your-privacy"
+                  }'>
+                   {{ _('Protect your privacy') }}
+                </a>
+              </li>
+              <li>
+                <a href="{{ url('questions.list_by_topic', topic_slug='manage-preferences-and-add-ons') }}"
+                   data-event-name="link_click"
+                   data-event-parameters='{
+                    "link_name": "main-menu.question-list",
+                    "link_detail": "manage-preferences-and-and-ons"
```

Comment:
Do we want to use `customize-settings-and-preferences` instead, since I think that's what we'll eventually arrive at right?